### PR TITLE
Expanded detail in RFLink TCP mode

### PR DESCRIPTION
**Pull request by Genestealer**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

### CODE REVIEW

**Comment by Genestealer** [comment]
Referred to PR in HASS Forums: 
[Project: RFLink to MQTT using ESP8266 and arduino IDE](https://community.home-assistant.io/t/arduino-nano-rflink/13926/2)
[Arduino Nano Rflink](https://community.home-assistant.io/t/project-rflink-to-mqtt-using-esp8266-and-arduino-ide/7131/9)

File: source/_components/rflink.markdown [hunk 1]
```diff
@@ -60,7 +60,11 @@ To expose the usb/serial interface over TCP on a different host (Linux) the foll
 $ socat /dev/ttyACM0,b57600 TCP-LISTEN:1234,reuseaddr
 ```
 
-Other methods of exposing the serial interface over TCP are possible (eg: ESP8266 or using Arduino Wifi shield). Basically the serial stream should be directly mapped to the TCP stream.
+Other methods of exposing the serial interface over TCP are possible (eg: ESP8266 or using Arduino Wifi shield). Essentially the serial stream should be directly mapped to the TCP stream.
+
+Tested with Wifi serial bridge [esp-link V2.2.3](https://github.com/jeelabs/esp-link/releases/tag/v2.2.3) running on a NodeMCU (ESP8266 Wifi module) with ESP8266 TXD0 (pin D10) and RXD0 (pin D9) connected to Arduino MEGA 2560 RX (Pin 2) and TX (Pin 3) respectively. 
+Warning: due to different logical levels, a voltage level shifter is required between the 3.3V NodeMCU and 5V Arduino MEGA 2560 pins.
```
Comment (Landrash) [inline source/_components/rflink.markdown]: You'll need to update your formating if you want to separate these sentences.

Comment (Landrash) [inline source/_components/rflink.markdown]: Due with capital letter.

**Comment by Genestealer** [comment]
@Landrash: Updated as per your comments.

**Comment by Landrash** [comment]
Spiced up your formatting a bit further. Is this live now or is it awaiting a PR in the main repo?
If so mention when it's live and I'll merge this.

**Comment by Genestealer** [comment]
@Landrash Should be ready for merge.

**Pull request merged**